### ISSUE TITLE
multiple dithers error

### ISSUE THREAD

**Issue by drgmk** [spacetelescope/pandeia-coronagraphy#9]
Folks,

I think if a bug has been introduced by the newer version of pandeia.engine (1.2). I've been running the various notebooks here, but always get the error
```UnsupportedError: Multiple dithers for imaging is not yet supported```
when the calculation is attempted (i.e. in calculate_batch).

Installing pandeia.engine-1.1.1 seems to solve this issue, and with 1.2 I can successfully run the quickstart notebook from pandeia-tutorials, so my installs seem OK.

I'm not at all familiar with the inner workings on pandeia, so I haven't been able to figure out what the problem actually is and don't have any suggestions sorry!

cheers
Grant

**Comment by mperrin** [comment]
Yes, this is a known issue! Haven't had time to deal with it yet, apologies. But it's on the to do list for "sometime soon"
